dspace-api/src/main/java/org/dspace/content/RelationshipServiceImpl.java

```diff
@@ -368,8 +379,8 @@ private boolean isRelationshipValidToDelete(Context context, Relationship relati
     private boolean checkMinCardinality(Context context, Item item,
                                         Relationship relationship,
                                         Integer minCardinality, boolean isLeft) throws SQLException {
-        List<Relationship> list = this
-            .findByItemAndRelationshipType(context, item, relationship.getRelationshipType(), isLeft);
+        List<Relationship> list = this.findByItemAndRelationshipType(context, item, relationship.getRelationshipType(),
+                                                                     -1, -1, isLeft);
```

Comment:
Why do you change the call here? Both calls should be equal and the previous one is shorter, or am I missing something?

Reply:
You're not missing anything I believe.  We're just invoking the paginated impl of the findByItemAndRelationshipType() method with the fallback limit and offset params.